### ISSUE TITLE
Need a view behind picker to block user action

### ISSUE THREAD

**Issue by onsensei** [beefe/react-native-picker#327]
Should have a view behind picker to block user action, because in most case user should finish the task in picker before do other things.

So, this module should have props to set
- `viewPartitionBackgroundColor` (I call this view as `ViewPartition`)
- `onPressViewPartition` to tell app when user tap on it, with this developer can implement something like in `onPickerCancel` or do other things

**Comment by beausmith** [comment]
Agreed. This feature request seems like a commonly desired feature. 

On iOS, I can display a modal which appears under the picker. However on Android, the picker displays below the modal, thus the whole app becomes unusable as the modal is blocking all clicks.

**Comment by raginisahu** [comment]
could you please give me a example.
